### DISCUSSION TITLE
Simplify Installation with a One-Command Docker Setup

### DISCUSSION THREAD

**Discussion by Livog** [coollabsio/coolify#4121] [Feature Requests]
Hi Coolify Team,

Love what you’re building and thank you for your hard work!

For someone with limited Docker experience, I would like to suggest easier installation steps. I found the best example for this to be EasyPanel.

Their installation steps are as follows:

1. **Install Docker Desktop:**

2. **Run this command:**

   ```bash
   docker run --rm -it \
     -v /etc/easypanel:/etc/easypanel \
     -v /var/run/docker.sock:/var/run/docker.sock:ro \
     easypanel/easypanel setup
   ```

3. **Done:** It will even finds your IP automatically (though you might need to expose port 3000 in your router).

Implementing a similar one-command Docker setup for Coolify would make it much easier for Docker newbies and significantly lower the barrier to entry. Users could simply install Docker Desktop and run a single script to get started.

Maybe this is already possible and I've got skill issue, if so please feel free to point me in the right direction.

Thanks for considering!

**Comment by peaklabs-dev** [comment]
For most operating systems you can run our one-click install script. A Docker command like this is not possible because our service does not only run on 1 container and also use mutliple compose files. You can follow the [manual installation instructions](https://coolify.io/docs/installation#2-manual-installation) to install Coolify manually. Also, it is intended that we use multiple compose files so that we have easier maintenance of the base compose and do not duplicate code while still having a compose file for each environment.
